catroid/src/main/res/layout/fragment_user_defined_brick_list.xml

```diff
@@ -41,14 +41,31 @@
         android:cacheColorHint="#00000000"
         android:divider="@android:color/transparent" />
 
+    <LinearLayout
+        android:id="@android:id/empty"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:clickable="true"
+        android:focusable="true"
+        android:layout_weight="1">
+
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:gravity="center"
+            android:text="@string/brick_user_defined_list_empty"
+            android:textSize="?attr/x_large" />
+
+    </LinearLayout>
+
     <ImageButton
         android:id="@+id/button_add_user_brick"
         android:layout_width="match_parent"
         android:layout_height="@dimen/actionbar_height"
-        android:layout_weight="0"
         android:layout_gravity="center|bottom"
+        android:layout_weight="0"
         android:background="@drawable/button_background_selector"
         android:src="@drawable/ic_plus"
-        tools:ignore="ContentDescription"/>
+        tools:ignore="ContentDescription" />
```

Comment:
Could you please revert the line changes that do not make any difference and only commit the new code? Everything else looks good to me